src/OpenTelemetry.AutoInstrumentation.Native/integration_loader.cpp

```diff
@@ -98,8 +108,45 @@ void LoadIntegrationsFromStream(std::istream& stream, std::vector<IntegrationMet
 
 namespace
 {
+    bool InstrumentationEnabled(const WSTRING name, const std::vector<WSTRING>& enabledIntegrationNames, const std::vector<WSTRING>& disabledIntegrationNames)
```

Comment:
This line is pretty long, suggestion: let's keep putting each parameter on its own line.